addons/medical_treatment/initSettings.sqf

```diff
@@ -125,6 +125,51 @@
     true
 ] call CBA_settings_fnc_init;
 
+[
+    QGVAR(requiredWoundStatePAK),
+    "LIST",
+    [LSTRING(RequiredWoundStatePAK_DisplayName), LSTRING(RequiredWoundStatePAK_Description)],
+    [ELSTRING(medical,Category), LSTRING(SubCategory_Treatment)],
+    [[0, 1, 2], [LSTRING(RequiredWoundStatePAK_Open), LSTRING(Bandaged), LSTRING(RequiredWoundStatePAK_Stitched)], 0],
+    true
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(allowPainPAK),
+    "LIST",
+    [LSTRING(AllowPainPAK_DisplayName), LSTRING(AllowPainPAK_Description)],
+    [ELSTRING(medical,Category), LSTRING(SubCategory_Treatment)],
+    [[0, 1], [ELSTRING(common,No), ELSTRING(common,Yes)], 1],
+    true
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(allowFracturesPAK),
+    "LIST",
+    [LSTRING(AllowFracturesPAK_DisplayName), LSTRING(AllowFracturesPAK_Description)],
+    [ELSTRING(medical,Category), LSTRING(SubCategory_Treatment)],
+    [[0, 1], [ELSTRING(common,No), ELSTRING(common,Yes)], 1],
+    true
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(minBloodPAK),
+    "SLIDER",
+    [LSTRING(MinBloodPAK_DisplayName), LSTRING(MinBloodPAK_Description)],
+    [ELSTRING(medical,Category), LSTRING(SubCategory_Treatment)],
+    [0, 1, 0.85, 0],
+    true
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(minHeartRatePAK),
+    "SLIDER",
+    [LSTRING(MinHeartRatePAK_DisplayName), LSTRING(MinHeartRatePAK_Description)],
+    [ELSTRING(medical,Category), LSTRING(SubCategory_Treatment)],
+    [0, 80, 40, 0],
+    true
+] call CBA_fnc_addSetting;
```

Comment:
For percentage slider set the 5th argument in `_valueInfo` to `true`.

Reply:
Right, my mistake. 🤦

Reply:
Resolved in https://github.com/anden3/ACE3/pull/4 -- Will request review once merged. Thanks.

Reply:
@mharis001 Merged. Thank you.